app/components/kb/relevant-articles.vue

```diff
@@ -2,93 +2,74 @@
     <div>
         <slot name="title">
             <h4>
-            {{ $t("relevantArticles") }} 
-                <hr></hr>
+            {{ t("relevantArticles") }}
+                <hr/>
             </h4>
         </slot>
-        <div class="loading" v-if="loading"><i class="fa fa-cog fa-spin fa-lg" ></i> {{ $t("loading") }}...</div>
+        <div class="loading" v-if="loading"><i class="fa fa-cog fa-spin fa-lg" ></i> {{ t("loading") }}...</div>
         <ul>
             <li v-for="article in articles" class="mb-1">
                 <a class="link-dark fs-6" :href="`${articleUrl(article)}`">{{lstring(article.title, $locale)}}</a>
```

Comment:
I think `$locale` is legacy and should come from the ```const { locale } = useI18n()```
```suggestion
                <a class="link-dark fs-6" :href="`${articleUrl(article)}`">{{lstring(article.title, locale)}}</a>
```

Reply:
done.